pandas/tests/io/parser/test_parse_dates.py

```diff
@@ -1512,18 +1476,15 @@ def test_generic(all_parsers):
     parser = all_parsers
     data = "year,month,day,a\n2001,01,10,10.\n2001,02,1,11."
 
-    with tm.assert_produces_warning(FutureWarning, check_stacklevel=False):
-        result = parser.read_csv(
-            StringIO(data),
-            header=0,
-            parse_dates={"ym": [0, 1]},
-            date_parser=lambda y, m: date(year=int(y), month=int(m), day=1),
-        )
-    expected = DataFrame(
-        [[date(2001, 1, 1), 10, 10.0], [date(2001, 2, 1), 1, 11.0]],
-        columns=["ym", "day", "a"],
+    def parse_function(yy, mm):
+        return [date(year=int(y), month=int(m), day=1) for y, m in zip(yy, mm)]
+
+    parser.read_csv(
```

Comment:
Is it not worth asserting the result of this call anymore?

Reply:
No, this was accidental, thx for noticing